modules/phase_field/examples/measure_interface_energy/1Dinterface_energy.i

```diff
@@ -0,0 +1,194 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+  nx = 100
+  xmax = 100
+  xmin = 0
+  elem_type = EDGE
+[]
+
+[GlobalParams]
+[]
+
+[AuxVariables]
+  [./local_energy]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+[]
+
+[AuxKernels]
+  [./local_free_energy]
+    type = TotalFreeEnergy
+    variable = local_energy
+    kappa_names = kappa_c
+    interfacial_vars = c
+  [../]
+[]
+
+[Variables]
+  [./c]
+    order = FIRST
+    family = LAGRANGE
+    scaling = 1e1
+    [./InitialCondition]
+      type = RampIC
+      variable = c
+      value_left = 0
+      value_right = 1
+    [../]
+  [../]
+  [./w]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+[]
+
+[Kernels]
+  [./c_res]
+    type = SplitCHParsed
+    variable = c
+    f_name = F
+    kappa_name = kappa_c
+    w = w
+  [../]
+  [./w_res]
+    type = SplitCHWRes
+    variable = w
+    mob_name = M
+  [../]
+  [./time]
+    type = CoupledTimeDerivative
+    variable = w
+    v = c
+  [../]
+
+[]
+
+[Functions]
+  [./Int_energy]
+    type = ParsedFunction
+    vals = 'total_solute Cmin Cmax Fleft Fright volume'
+    value = ((total_solute-Cmin*volume)/(Cmax-Cmin))*Fleft+(volume-(total_solute-Cmin*volume)/(Cmax-Cmin))*Fright
+    vars = 'total_solute Cmin Cmax Fleft Fright volume'
+  [../]
+[]
+
+[Materials]
+  [./consts]
+    type = GenericConstantMaterial
+    block = 0
+    prop_names  = 'kappa_c M'
+    prop_values = '25      150'
+  [../]
+  [./Free_energy]
+    type = DerivativeParsedMaterial
+    f_name = F
+    function = c^2*(c-1)^2
+    args = c
+    derivative_order = 2
+    block = 0
+  [../]
+[]
+
+[Postprocessors]
+  # The total free enrgy of the simulation cell to observe the energy reduction.
+  [./total_free_energy]
+    type = ElementIntegralVariablePostprocessor
+    variable = local_energy
+  [../]
+
+  # for testing we also monitor the total solute amount, which should be conserved,
+  # gives Cavg in % for this problem.
+  [./total_solute]
+    type = ElementIntegralVariablePostprocessor
+    variable = c
+  [../]
+  # Get simulation cell size (1D volume) from postprocessor
+  [./volume]
+    type = ElementIntegralMaterialProperty
+    block = 0
+    mat_prop = 1
+  [../]
+  # Find concentration in each phase using max and min
```

Comment:
No, this is _wrong_! We do not know a priori if the concnetrations are the min and max, or the max and min respectively! You need to be consistent with the Fleft and Fright PPs! Use SideAverageValue.